packages/frontend/src/utils/file.ts

```diff
@@ -7,3 +7,8 @@ export const saveLocalFile = (data: Blob, fileName: string): void => {
   a.click();
   window.URL.revokeObjectURL(url);
 };
+
+export const imageFullPath = (filename: string): string => {
```

Comment:
Same note as above with getGenericSettingValueRealized -- I think the previous architecture is preferable (where the image's full url is defined by the server).

Reply:
Fixed